### PR TITLE
Fix Versatility Adept and imporve description

### PR DESCRIPTION
**Pull request by HIEROT**

### CODE REVIEW

File: SolastaUnfinishedBusiness/Subclasses/Builders/EldritchVersatility.cs [hunk 1]
```diff
@@ -80,11 +80,17 @@ public enum PointUsage
             .Setup(InvocationPoolTypeCustom.Pools.EldritchVersatilityPool, 1, true)
             .AddToDB();
 
+    public static readonly FeatureDefinition FeatureEldritchVersatilityGrantPoolAndSwitch = 
+        FeatureDefinitionBuilder
+            .Create($"Feature{Name}GrantPoolAndSwitch")
+            .SetGuiPresentation(Category.Feature, hidden: true)
```
Comment (ThyWoof) [inline SolastaUnfinishedBusiness/Subclasses/Builders/EldritchVersatility.cs:86]: no need for a GUI Presentation here. Use SetGuiPresentationNoContent(true)

**Review by ThyWoof — CHANGES_REQUESTED** [review]
check minor change request

**Comment by HIEROT** [comment]
@ThyWoof done

**Review by ThyWoof — APPROVED** [review]

**Comment by HIEROT** [comment]
please revert the last commit as it was a mistake

**Pull request merged**